docs/third-party-software-citations.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Third party software citations
 parent: Library
-nav_order: 9
+nav_order: 10
```

Comment:
This hard-coded nav order system we have does feel less graceful than I would like, but :shrug:

Reply:
yeah it's not ideal, but we're not dealing with Gravity's Rainbow here or anything